app/model/sdl/Abstract/Model.js

```diff
@@ -666,7 +677,24 @@ SDL.SDLModel = Em.Object.extend({
         var sdl_stream = SDL.SDLController.getApplicationModel(
           appID
         ).navigationStream;
-
+        
+        console.log(' set params from VideoConfig');
```

Comment:
@LuxoftAKutsan please don't use console output directly. We have a dedicated class for logging:
```
Em.Logger.log(`Set params from VideoConfig`);
```

Reply:
72bd13d